### DISCUSSION TITLE
Gesamtertrag Gestern

### DISCUSSION THREAD

**Discussion by FritzFritzel** [tbnobody/OpenDTU#1255] [Ideas]
Ich würde mir wünschen, dass der "Gesamtertrag Gestern" angezeigt wird.
Oder vielleicht sogar eine kleine Historie für die letzten 7 Tage als Ringspeicher.

**Comment by Omega13x** [comment]
Die DTU zeigt nur an was der WR hergibt. Wenn es etwas luxuriöser sein soll, muss man etwas dafür tun.
Kleines Beispiel:
![image](https://github.com/tbnobody/OpenDTU/assets/123419851/10e0b44f-abfe-4dc9-bb41-23cce174339b)
![image](https://github.com/tbnobody/OpenDTU/assets/123419851/f3dfd193-b8a3-4b73-b406-3d6e51ff9e7b)

**Reply by FritzFritzel** [reply]
Man bräuchte doch nur täglich um Mitternacht den "Gesamtertrag Heute" in den "Gesamtertrag Gestern" zu kopieren.
Das ist halt für Leute, die nur eine DTU haben und keinen ioBroker o.ä.

**Reply by DPO99** [reply]
Aber da geht's ja schon los...heute gestern 7 Tage oder gleich einen monat oder Jahr? Die dtu ist mit dem kleinen ESP auf das auslesen und steuern ausgelegt. Historische Daten muss man sich eben rausziehen und da braucht es nicht zwingend einen iobroker.

**Comment by huste511** [comment]
Über das gestern kann man sich noch streiten ... aber das heute sollte einem schon angezeigt werden.

Komme ich kurz nach Dämmerung nach Hause und schaue auf das Diplay steht da: "Heute: 0Wh" ;-(

Die anderen Wechselrichterinformationen werden auch Nachts im dem GUI vorgehalten, wenn die MWR aus sind.

**Reply by tbnobody** [reply]
Der tägliche Ertrag wird beim neu-start des Wechselrichters auf 0 gesetzt. Wenn es heute bei dir aufgrund mangelnder Sonneneinstrahlung so dunkel war dass der Inverter aus ging, danach nochmals die Sonne kam das er wieder aktiv wurde war der Wert ggf. 0. (Das war nun mal der letzte Wert der vom Inverter gelesen wurde)

**Reply by huste511** [reply]
Stimmt ... die MWR sind laut Ereignisanzeige um 20:30 noch mal angelaufen, haben aber nichts mehr geliefert.

btw: Empfehlung für Forum für div. Hardwarefragen?
